packages/node/src/lib/subscribers/subscribers.ts

```diff
@@ -48,6 +48,12 @@ export class Subscribers extends WithHttp implements ISubscribers {
     });
   }
 
+  async create(subscriberId: string, data: ISubscriberPayload) {
```

Comment:
@gitstart this is not needed. There is the identify function that does that. 
No changes are needed in the node sdk.

Reply:
The function has been removed @ainouzgali, thanks for the feedback.